etna/packages/etna-js/jest.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
I extracted this out of the package.json#jest config object once it started to get bigger.  This would allow us to add comments and piece together / share config in the future easier than it being part of the package.json payload.